gcp_variant_transforms/vcf_to_bq_common.py

```diff
@@ -115,3 +118,19 @@ def write_headers(merged_header, file_path):
   """Writes a PCollection of ``VcfHeader`` to location ``file_path``."""
   _ = (merged_header | 'WriteHeaders' >>
        vcf_header_io.WriteVcfHeaders(file_path))
+
+
+def _raise_error_on_unrecognized_flags(pipeline_args):
+  # type: (List[str]) -> None
+  """Raises an error if there are unrecognized flags."""
+  options = pipeline_options.PipelineOptions(pipeline_args).get_all_options()
+  for flag in _get_flag_names(pipeline_args):
+    if not any(str.startswith(option, flag) for option in options):
```

Comment:
nit: any reason to not use `option.startswith(flag)`? it's slightly more readable.

Reply:
Right. Updated.